src/controllers/adminController.ts

```diff
@@ -1,42 +1,73 @@
+        }
+        catch (err) {
+            return res.send({ status: 400, data: [], message: "Something went wrong" });
+        }
+    }
+
+    public static async filterRequestList(req: Request, res: Response):Promise<Response> {
+        try {
+            const { organisation,filterType,requestStatus,user,date,page,limit } = req.body;
+            // console.log(organisation,filterType,requestStatus,user,date);
+            const obj={orgUniqName:organisation,filterType,requestStatus,user,date,page,limit};
+            const result = await AdminServices.getFilterRequestList(obj);
+            return res.send({ status: 200, data: result, messsage: "Success" });
+        }
+        catch (err) {
+            return res.send({ status: 400, data: [], message: "Something went wrong" });
+        }
+    }
 
-            const {orgUniqName,userEmail,availedAtDay,availedAtMonth,availedAtYear,rejectionReason}=req.body;
-            const requestStatus="accept";
-            const oldRequestStatus='pending';
-            const result=await AdminServices.updateRequestService(orgUniqName,userEmail,requestStatus,availedAtDay,availedAtMonth,availedAtYear,rejectionReason,oldRequestStatus);
-            res.send(result);
+    public static async acceptRequest(req: Request, res: Response):Promise<Response> {
+        try {
+            const { organisationUserEmail, organisation, availedAt } = req.body;
+            const requestStatus = "Approved";
+            const oldRequestStatus = 'Pending';
+            const rejectionReason = '';
+            const obj={orgUniqName:organisation,userEmail:organisationUserEmail,requestStatus, availedAt, rejectionReason, oldRequestStatus}
+            const result = await AdminServices.updateRequestService(obj);
+            if (result) return res.send({ status: 200, message: "Requested Accepted" });
+            else return res.send({ status: 400, message: "Failed to Accepted" });
         }
-        catch(err){
-            res.send({success:false});
+        catch (err) {
+            return res.send({ status: 400, message: "Failed to Accepted" });
         }
     }
 
-    public static async rejectRequest (req:Request,res:Response){
-        try{
-            const {orgUniqName,userEmail,availedAtDay,availedAtMonth,availedAtYear,rejectionReason}=req.body;
-            const requestStatus="reject";
-            const oldRequestStatus='pending';
-            const result=await AdminServices.updateRequestService(orgUniqName,userEmail,requestStatus,availedAtDay,availedAtMonth,availedAtYear,rejectionReason,oldRequestStatus);
-            res.send(result);
-        }
-        catch(err){
-            res.send({success:false});
-        }
-    }    
+    public static async rejectRequest(req: Request, res: Response):Promise<Response> {
+        try {
+            const { organisationUserEmail, organisation, availedAt, requestRejectionReason } = req.body;
```

Comment:
req.body should accept snake case only , here can use { email, availed_at, rejection_reason}